coralproject/talk
# [CORL-664] Threshold moderate new commenters

## What does this PR do?

This is the same changes @tessalt previously wrote with two minor changes.

- Updates the old branch to latest master with all the fragment and style changes around the moderation/config area
- Replaces the prior counting calculation with a simple lookup from the user's cached `commentCounts` introduced in CORL-688

If there is a `don't merge` label on this PR, it's due to the fact that `CORL-688` likely hasn't been merged yet.

## How do I test this PR?

- Login as an admin/mod
- Head to the Moderation config and enable new commenter approval feature
- Default approves is likely set to 2
- Create a new commenter and sign in
- Make a couple comments on the stream
- See that they get sent for moderation
- Check back in the admin side with your mod account in the pending queue
- Approve 2 of their comments (or whatever your approval threshold setting was)
- Go back to the stream and post a comment as the commenter
- Their comments should be showing up in stream and not be held for moderation

NOTE: make sure you turn auto pre-mod off on the stream you choose to test with
